iis/datadog_checks/iis/iis.py

```diff
@@ -55,62 +55,72 @@ def get_iishost(self, instance):
         return "iis_host:{}".format(self.normalize(iis_host))
 
     def check(self, instance):
-        sites = instance.get('sites')
-        if sites is None:
-            expected_sites = set()
-        else:
-            expected_sites = set(sites)
+        sites = instance.get('sites', [])
+        expected_sites = set(sites)
```

Comment:
Note that this isn't completely the same. `sites` could be in `instance` but `None`, that would change the behavior. I don't think it matters much in this case though.

Reply:
`sites = instance.get('sites', []) or []` ?